Graphics/DirectX/DirectXEngine.cpp

```diff
@@ -342,6 +360,12 @@ void leap::graphics::DirectXEngine::Draw()
 
 	if (m_pCamera)
 	{
+		if (!m_DebugDrawings.GetIndexBuffer().empty())
+		{
+			m_pDebugRenderer->LoadMesh(m_DebugDrawings);
+			m_DebugDrawings = {}; // Reset debug renderer
```

Comment:
Maybe create a reset/clear function for CustomMesh. Now every frame the vector will get recreated and resized along the frame calls. Problem with not recreating is that the vector stays large if many calls happend.